seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -4556,8 +4556,8 @@
         "",
         "// Only change code below this line",
         "function updateRecords(id, prop, value) {",
-        "",
-        "",
+        "  ",
+        "  ",
```

Comment:
@Bouncey they're helpful for campers. 

You may verify other challenge seeds. Almost all of the JavaScript challenge seeds have spaces. They actually helpful. :smile: 

> A minor improvement but it's worth it.